cli/package.json

```diff
@@ -34,8 +34,10 @@
     "test:debug": "node --inspect-brk $(npm bin)/jest --watchAll --no-cache --runInBand"
   },
   "dependencies": {
+    "@types/cli-table": "^0.3.0",
```

Comment:
Can we move this to a dev dependency? I'm building the code before publishing, so the types packages are only needed during development.

Reply:
Done!